src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -20,25 +25,39 @@ const StyledChip = styled(Chip)({
   },
 });
 
-export const ChannelClaimTableRow = ({ claimedChannel }: ChannelClaimTableRowProps) => {
+export const ChannelClaimTableRow = ({ claimedChannel, channelType }: ChannelClaimTableRowProps) => {
   const { t } = useTranslation();
+  const channelId = claimedChannel.channelClaim.channel;
+
+  const isPublisherChannel = channelType === 'publisher';
+
+  const publisherQuery = useFetchPublisher(channelId, isPublisherChannel);
```

Comment:
kanskje vært mer naturlig å ikke sette `channalId` her når vi vet at det ikke er en publisher?
Noe som skurrer litt hos meg med å kalle `useFetchPublisher` med en `id` vi vet ikke peker til en utgiver, for så å sette et flagg som sier at den ikke skal hente data 😅 

```suggestion
  const publisherQuery = useFetchPublisher(isPublisherChannel ? channelId : "");
```